### PR TITLE
Jax cli

### PR DESCRIPTION
**Pull request by priyanshuone6**
- Add `__init__`
- Add --help in `pybamm_install_jax`
- Add an option to force install compatible versions of jax and jaxlib if already installed using cli

### CODE REVIEW

File: pybamm/util.py [hunk 1]
```diff
@@ -364,23 +365,50 @@ def is_jax_compatible():
...
     elif importlib.util.find_spec("jax") is not None:
-        if not is_jax_compatible():
+        if not args.force and not is_jax_compatible():
             raise ValueError(
-                "Jax is already installed but the installed version of jax or jaxlib is not supported by PyBaMM",  # noqa: E501
+                f"""
+Jax is already installed but the installed version of jax or jaxlib is not supported by PyBaMM.
+
+You can force install compatible versions of jax ({JAX_VERSION}) and jaxlib ({JAXLIB_VERSION}) using the following command:
+pybamm_install_jax --force""",  # noqa: E501
```
Comment (tinosulzer) [inline pybamm/util.py:402]: Can you use single quotes here instead, like https://github.com/pybamm-team/PyBaMM/blob/fb6ebaa12900d227d3a247afdb7acfa9f5b7c62d/pybamm/simulation.py#L1000?

File: pybamm/util.py [hunk 2]
```diff
@@ -364,23 +365,50 @@ def is_jax_compatible():
...
+    optional arguments:
+    -h, --help   show help message
+    -f, --force  force install compatible versions of jax and jaxlib
+    """
+    parser = argparse.ArgumentParser(description="Install jax and jaxlib")
+    parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        help=f"force install compatible versions of jax ({JAX_VERSION}) and jaxlib ({JAXLIB_VERSION})",  # noqa: E501
```
Comment (tinosulzer) [inline pybamm/util.py:385]: can you make this 2 lines?

**Review by tinosulzer — APPROVED** [review]
Thanks, just some style comments

**Comment by tinosulzer** [comment]
What are the lines that lost coverage?

**Comment by priyanshuone6** [comment]
Not sure on why this is happening. Might be due to adding few init files?

**Comment by Saransh-cpp** [comment]
Looks like adding an `__init__.py` to the `models/full_battery_models/lithium_metal` directory exposed [`dfn.py`](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/models/full_battery_models/lithium_metal/dfn.py) to coverage.

**Comment by tinosulzer** [comment]
Does there have to be an `__init__.py` in that folder right now? Can you open an issue to add the test?

**Comment by priyanshuone6** [comment]
Should I remove the init file from lithium_metal?

**Comment by tinosulzer** [comment]
Yes, thanks

**Pull request merged**